src/components/FeatureViews/Scheduler/SessionView.js

```diff
@@ -304,33 +304,24 @@ const SessionView = () => {
                     {/* <InstructorSchedule instructorID={instructor_id} /> */}
                 </Grid>
             </Grid>
-            <Grid
-                className='session-detail-action-control'
-                container
-                direction='row'
-                justify='flex-end'
-            >
+            <Grid container direction='row' justify='flex-end' spacing={1}>
                 <Grid item>
-                    <Button
-                        className='button'
-                        color='secondary'
+                    <ResponsiveButton
                         component={NavLink}
-                        to={`/registration/course/${course_id}`}
+                        to={`/courses/class/${course_id}`}
                         variant='outlined'
                     >
                         Course Page
-                    </Button>
+                    </ResponsiveButton>
                 </Grid>
-                <Grid item></Grid>
                 <Grid item>
-                    <Button
-                        className='editButton'
+                    <ResponsiveButton
                         color='primary'
                         onClick={handleEditToggle(true)}
-                        variant='outlined'
+                        variant='contained'
```

Comment:
A parent shouldn't be able to see the Reschedule button. Suggest adding some account validation.